console-src/src/components/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+import colors from "../../theme/colors";
+import { Link } from "react-router-dom";
+
+const LinkButton = styled(Link)`
+  border-style: none;
+  outline: none;
+  cursor: pointer;
+  color: white;
+  width: 150px;
+  min-width: max-content;
+  height: 50px;
```

Comment:
What would happen when text is wrapped to the next line? Do we need to prevent it and add ellipsis?

Reply:
Maybe yeah, I have shamelessly copy pasted these styles from the webapp but I don't see why we can't improve on them 👍

Reply:
Actually the `width` property is not respected when text goes above 150px, because <a> generated by `Link` is an `inline` element.

I think I will just remove the `width` here, and try to keep button texts short, I think it's important that whole button text are visible

Reply:
I have done it here: 053eccaebf699e3f8c2ef19359d4f8a831433e08 I think it's the best approach for this

Reply:
awesome, thanks